test/es6/rlexe.xml

```diff
@@ -1480,4 +1480,10 @@
     <baseline>supersetter.baseline</baseline>
   </default>
 </test>
+  <test>
+    <default>
+      <files>bug_OS16649604.js</files>
+      <compile-flags> -maxinterpretcount:1 -maxsimplejitruncount:1 -off:simplejit -args summary -endargs</compile-flags>
```

Comment:
> -maxinterpretcount:1 -maxsimplejitruncount: [](start = 21, length = 44)

You can remove these two flags - 
these two flags will be added by rl for the interpreted variant of the tests. For dynapogo, we run with -forcenative, and these flags won't matter.